base_layer/wallet_ffi/src/lib.rs

```diff
@@ -1872,6 +1897,43 @@ pub unsafe extern "C" fn wallet_test_complete_sent_transaction(
     }
 }
 
+/// This function checks to determine if a TariCompletedTransaction was originally a TariPendingOutboundTransaction
+///
+/// ## Arguments
+/// `wallet` - The TariWallet pointer
+/// `tx` - The TariCompletedTransaction
+/// `error_out` - Pointer to an int which will be modified to an error code should one occur, may not be null. Functions
+/// as an out parameter.
+///
+/// ## Returns
+/// `bool` - Returns if the transaction was originally sent from the wallet
+#[no_mangle]
+pub unsafe extern "C" fn wallet_check_originating_completed_transaction(
```

Comment:
This is a confusing function name seeing as it returns bool.

I propose: `wallet_is_completed_transaction_outbound(...)`

Reply:
Done